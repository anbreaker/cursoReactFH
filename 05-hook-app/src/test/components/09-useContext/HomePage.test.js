const {mount} = require('enzyme');
const {HomePage} = require('../../../components/09-useContext/HomePage');
const {UserContext} = require('../../../components/09-useContext/UserContext');

describe('Pruebas en <HomePage /> ', () => {
  const user = {
    name: 'anbreaker',
    email: 'anbreaker@gmail.com',
  };

  const wrapper = mount(
    <UserContext.Provider value={{user}}>
      <HomePage />
    </UserContext.Provider>
  );

  test('debe de mostrarse correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
