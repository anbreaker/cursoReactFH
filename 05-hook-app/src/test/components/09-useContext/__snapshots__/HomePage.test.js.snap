// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HomePage />  debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HomePage
  </h1>
  <hr />
  <pre>
    {
   "name": "anbreaker",
   "email": "anbreaker@gmail.com"
}
  </pre>
</div>
`;
